### PR TITLE
Inprogress

### PR DESCRIPTION
**Pull request by dotNATE**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,5 +15,24 @@ <h4 class="nav-logo">nathaniel</h4>
 				<a class="nav-link" href="#contact">Contact</a>
 			</div>
 		</nav>
+		<main>
+			<div class="main-content">
+				<img />
```
Comment (SobiaF) [inline index.html]: I'm guessing this is a placeholder for an img?

File: index.html [hunk 2]
```diff
@@ -14,5 +15,24 @@ <h4 class="nav-logo">nathaniel</h4>
 				<a class="nav-link" href="#contact">Contact</a>
```
Comment (SobiaF) [inline index.html]: will you be adding more here or just contacts? 🤔

**Review by SobiaF — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (dotNATE) [reply index.html]: just projects and contact for now, maybe more once site starts to grow

File: index.html (hunk 1 above)
Comment (dotNATE) [reply index.html]: mortified that I left this out, filling in now!

**Pull request merged**